test/foods-response-test.js

```diff
@@ -0,0 +1,168 @@
+const assert = require('chai').assert;
+const app = require('../server.js');
+const request = require('request');
+const Food = require('../lib/models/food');
+const food = new Food
+
+const environment = process.env.NODE_ENV || 'test';
+const configuration = require('../knexfile')[environment];
+const database = require('knex')(configuration);
+
+function truncate (done) {
+  database.raw('TRUNCATE foods RESTART IDENTITY CASCADE').then(() => {
+    return database.raw('TRUNCATE categories RESTART IDENTITY CASCADE');
+  }).then(() => {
+    return database.raw('TRUNCATE meals RESTART IDENTITY CASCADE');
+  }).then(() => {
+    done();
+  }).catch(done);
+}
+
+describe('Server', () => {
```

Comment:
Do we want to change the name of this outer describe block to better match the contents of this test?